source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1571,15 +1571,24 @@ message TransformationLoad {
 
   // Transformation that adds an OpLoad instruction from a pointer into an id.
 
-  // The result of the load instruction
+  // The result of the load instruction.
   uint32 fresh_id = 1;
 
-  // The pointer to be loaded from
+  // The pointer to be loaded from.
   uint32 pointer_id = 2;
 
+  // True if and only if the load should be atomic.
+  bool is_atomic = 3;
+
+  // Memory scope for the load. Ignored unless the load is atomic.
+  uint32 memory_scope = 4;
+
+  // The control masks define bitmask hints for function optimization.
+  uint32 memory_semantics = 5;
```

Comment:
Remove this. If `is_atomic` is true then we want the `TransformationLoad` class to always add an `OpAtomicLoad` instruction with memory semantics `None (Relaxed)` bitwise OR with either `WorkgroupStorage` or `UniformStorage`.

The code in the `TransformationLoad` class will choose `WorkgroupStorage` or `UniformStorage` by considering the storage class of the type of the instruction `pointer_id`.

Thus, the code in the `TransformationLoad` class does not require any memory semantics information from the protobuf.

Reply:
Sorry, please ignore this comment.